data/scopes/yealink-T43U.ini

```diff
@@ -0,0 +1,40 @@
+[metadata]
+scopeType = "model"
+displayName = "Yealink T43U"
+
+[data]
+cap_ldap_tls_blacklist =
+provisioning_url_filename = 
+tmpl_phone = 'yealink.tmpl'
```

Comment:
```suggestion
tmpl_phone = "yealink.tmpl"
```

Reply:
it's a problem in all Yealink scopes, I'll fix this in another commit 
```
tux@home:~/build/tancredi$ grep "'" data/scopes/*
data/scopes/yealink-T19P_E2.ini:tmpl_phone = 'yealink.tmpl'
data/scopes/yealink-T21P_E2.ini:tmpl_phone = 'yealink.tmpl'
data/scopes/yealink-T23G.ini:tmpl_phone = 'yealink.tmpl'
data/scopes/yealink-T27G.ini:tmpl_phone = 'yealink.tmpl'
data/scopes/yealink-T29G.ini:tmpl_phone = 'yealink.tmpl'
data/scopes/yealink-T40PG.ini:tmpl_phone = 'yealink.tmpl'
data/scopes/yealink-T41PS.ini:tmpl_phone = 'yealink.tmpl'
data/scopes/yealink-T42GS.ini:tmpl_phone = 'yealink.tmpl'
data/scopes/yealink-T43U.ini:tmpl_phone = 'yealink.tmpl'
data/scopes/yealink-T46GS.ini:tmpl_phone = 'yealink.tmpl'
data/scopes/yealink-T48GS.ini:tmpl_phone = 'yealink.tmpl'
data/scopes/yealink-T49G.ini:tmpl_phone = 'yealink.tmpl'
data/scopes/yealink-T53.ini:tmpl_phone = 'yealink.tmpl'
data/scopes/yealink-T54.ini:tmpl_phone = 'yealink.tmpl'
data/scopes/yealink-T56.ini:tmpl_phone = 'yealink.tmpl'
data/scopes/yealink-T57.ini:tmpl_phone = 'yealink.tmpl'
data/scopes/yealink-T58.ini:tmpl_phone = 'yealink.tmpl'
```